pkg/apis/clusterlink.net/v1alpha1/import.go

```diff
@@ -50,7 +50,9 @@ type ImportSpec struct {
 	TargetPort uint16 `json:"targetPort,omitempty"`
 	// Sources to import from.
 	Sources []ImportSource `json:"sources"`
-	// TODO: add LoadBalancingSpec.
+	// LBScheme is the load-balancing scheme to use (e.g., Random, Static, ECMP)
+	LBScheme string `json:"lbScheme"`
+	// TODO: Make LBScheme a proper type (when backwards compatibility is no longer needed)
```

Comment:
I don't see why moving the scheme definitions from `loadBalancer.go` into here will break anything.
But I can do it in a follow-up PR if you prefer.

Reply:
Yes, I believe a separate PR is more proper